pkg/importer/vddk-datasource_test.go

```diff
@@ -666,14 +666,14 @@ var _ = Describe("VDDK get block status", func() {
 		flagSetting := true
 		currentMockNbdFunctions.BlockStatus = func(length uint64, offset uint64, callback libnbd.ExtentCallback, optargs *libnbd.BlockStatusOptargs) error {
 			err := 0
-			len := uint32(MaxBlockStatusLength)
+			L := uint32(MaxBlockStatusLength)
```

Comment:
why use an upper case L instead of a lower case one?

Reply:
The linter dislikes shadowing built-in variables and functions such as `len`, `max`, etc.

The uppercase L is just my lack of originality

Reply:
Right it is just that normally upper cass variables indicate 'public' type variables. Which of course is not a thing inside a function, it is just confusing. I would just use lower case l and we can call it a day.

Reply:
Sounds good.